test/models/announcement_test.rb

```diff
@@ -1,37 +1,37 @@
-require 'test_helper'
+require "test_helper"
 
 class AnnouncementTest < ActiveSupport::TestCase
-  test 'can create an announcement with the proper info' do
-    assert_difference 'Announcement.count', +1 do
-      announcement_params = {title: 'NO School', body: 'WOOT WOOT', expiration_date: 2.days.from_now, created_by: 'richard'}
+  test "can create an announcement with the proper info" do
+    assert_difference "Announcement.count", +1 do
+      announcement_params = {title: "NO School", body: "WOOT WOOT", expiration_date: 2.days.from_now, created_by: "richard"}
```

Comment:
Line is too long. [124/80]<br>Space inside { missing.<br>Space inside } missing.